sandbox/libs/authn/src/main/java/org/opensearch/authn/realm/InternalRealm.java

```diff
@@ -109,7 +109,7 @@ protected AuthenticationInfo doGetAuthenticationInfo(AuthenticationToken token)
                 return sai;
             } else {
                 // Bad password
-                throw new IncorrectCredentialsException();
+                throw new IncorrectCredentialsException("Incorrect credentials");
```

Comment:
May be worthwhile to make a standardized error message at the top i.e. `INCORRECT_CREDENTIALS_MESSAGE = The incorrect credentials were provided.` Just a thought.